docs/maintenance/Pull_Requests.mdx

```diff
@@ -111,6 +108,22 @@ For preliminary reviews, be clear with what scale you're reviewing at: _"Reviewi
 For repeat reviews, be clear when it's something you missed earlier and/or there's new information.
 Don't apologize if the missed information was only made clear because of requested changes - this is part of the review process!
 
+### Approvals
+
+Upon completing your review, if the build is passing and you have no blockers, approve it on GitHub.
+Then:
+
+- If there isn't a `1 approval` label or existing approval, add the `1 approval` label
+- If there's already `1 approval` and/or it's been a week since the last request for changes, go ahead and squash merge
+  - For straightforward PRs that don't impact users, you can wait 3 business days instead
+
+We try to wait a week on PRs to give part-time reviewers a chance to get to it.
```

Comment:
I couldn't figure out how to phrase the idea succinctly. I.e. I want to make sure that reviewers who only have 1-2 days a week to review get a chance to see things.

This has been an issue for me in the past in other projects (not this one), where things get shoved through by 2 people in a small number of business days and I never got a chance to give input. 😞

Reply:
Maybe like this?
```suggestion
We try to wait a week on PRs to give reviewers who aren't active every day a chance to get to it.
```